Release/src/http/client/http_client_winhttp.cpp

```diff
@@ -244,7 +244,6 @@ class winhttp_request_context final : public request_context
     bool is_externally_allocated() const { return !m_body_data.is_internally_allocated(); }
 
     HINTERNET m_request_handle;
-    std::weak_ptr<winhttp_request_context>* m_request_context;
```

Comment:
@ras0219-msft  Can you comment on why this was a member before, or do you not remember? It looks like it could have only ever made a difference if _start_request_send was recursively reentered.